libs/wire-api/src/Wire/API/Routes/Public/Brig.hs

```diff
@@ -268,6 +274,66 @@ data Api routes = Api
         :> "list-prekeys"
         :> ReqBody '[JSON] QualifiedUserClients
         :> Post '[JSON] QualifiedUserClientPrekeyMap,
+    -- User Client API ----------------------------------------------------
+    -- TODO: another one?
```

Comment:
Forgotten TODO?

Reply:
That was already there, so I copy pasted it back. I'm not sure what it means.

Reply:
Looks like it sneaked in while previous refactoring: https://github.com/wireapp/wire-server/commit/37039b64291ac77be2666f6b74a1bcab39ce33cb#diff-4652702c562703881b30f146d018046eb3d53b00c4c1d94ab753943e9d54bc6aR503

Let's remove it.